### PR TITLE
Qualys scan csv

### PR DESCRIPTION
**Pull request by 37b**
Finally updated Qualys Scan CSV importer to read CVSS from existing values (though the data has to be massaged a little first.

Also included a minor fix for #6274

### CODE REVIEW

**Comment by 37b** [comment]
Fixing those tests >.<

**Review by damiencarol — APPROVED** [review]

File: dojo/fixtures/initial_surveys.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
       "modified": "2015-03-30T19:31:16Z",
       "optional": false,
       "order": 1,
-      "polymorphic_ctype": 18
+      "polymorphic_ctype": 77
```
Comment (StefanFl) [inline dojo/fixtures/initial_surveys.json:9]: Are the changes in this file related to the PR? If not, the changes should be reverted.

**Review by StefanFl — CHANGES_REQUESTED** [review]

**Review by StefanFl — CHANGES_REQUESTED** [review]
There are a lot of formatting changes in this PR. I can't see if there are any other changes hidden in these files. The PR should only contain changes that are necessary. Please revert all other changes. This might close the problems with flake8 as well.

**Comment by 37b** [comment]
> There are a lot of formatting changes in this PR. I can't see if there are any other changes hidden in these files. The PR should only contain changes that are necessary. Please revert all other changes. This might close the problems with flake8 as well.

I saw that - sorry about that. I have been using autopep8 with personal projects and I think some wires got crossed.

File: dojo/tools/qualys/csv_parser.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    report_findings = []
+
+    for row in csv_reader:
+        if row.get('Title') and row['Title'] != 'Title':
+            report_findings.append(row)
+
+    return report_findings
+
+
+def get_references(cve_list):
```
Comment (StefanFl) [inline dojo/tools/qualys/csv_parser.py:93]: There is a new mechanism for vulnerability ids like CVEs or GHSA's. You can fill them in a list called finding.unsaved_vulnerability_ids. They will all be showed in the UI and will all be clickable to show the content. I would prefer using this than building your own set of references.

File: unittests/tools/test_qualys_parser.py [hunk 3]
```diff
@@ -46,9 +46,51 @@ def test_parse_file_with_multiple_vuln_has_multiple_findings(self):
...
+        for finding in findings:
+            if finding.unsaved_endpoints[0].host == "10.98.57.180" and finding.title == "QID-105971 | EOL/Obsolete Software: Microsoft ASP.NET 1.0 Detected":
+
+                self.assertEqual(
+                    finding.severity, "Critical"
+                )
+                self.assertEqual(
+                    finding.cvssv3,
+                    "CVSS:3.0/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H/E:U/RL:U/RC:C")
+                self.assertEqual(
```
Comment (StefanFl) [inline unittests/tools/test_qualys_parser.py:94]: Why do you check the severity 2 times?

**Review by StefanFl — CHANGES_REQUESTED** [review]

**Review by StefanFl — APPROVED** [review]

**Pull request merged**